documentation/src/main/asciidoc/topics/rest_examples/rest_full_cache_detail.json

```diff
@@ -0,0 +1,48 @@
+{
+  "stats": {
+    "misses": -1,
+    "timeSinceStart": -1,
+    "timeSinceReset": -1,
+    "hits": -1,
+    "currentNumberOfEntries": -1,
```

Comment:
Those fields needs adjustment...The REST output is not camel cased, but lowercase with "_", example ```cache_manager_status```

Reply:
An easy way to do it is to instrument the object mapper like https://github.com/infinispan/infinispan/blob/0d5fe79599070c1619848ccb1815fc77355604ae/server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java#L71

Reply:
But this is the case in all the CacheV2 then, isn't it ?